sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/schemas/Sgv2TablesResource.java

```diff
@@ -185,14 +187,34 @@ public Response createTable(
       @ApiParam(value = "Name of the keyspace to use for the request.", required = true)
           @PathParam("keyspaceName")
           final String keyspaceName,
-      @ApiParam(required = true) @NotNull final TableAdd tableAdd,
+      @ApiParam(required = true) @NotNull final Sgv2TableAddRequest tableAdd,
       @Context HttpServletRequest request) {
     if (isAuthTokenInvalid(token)) {
       return invalidTokenFailure();
     }
-    logger.info("createTable() called for KS '" + keyspaceName + "'");
+    if (isStringEmpty(keyspaceName)) {
+      return jaxrsBadRequestError("keyspaceName must be non-empty").build();
+    }
+    final String tableName = tableAdd.getName();
+    if (isStringEmpty(keyspaceName)) {
+      return jaxrsBadRequestError("Table name must be non-empty").build();
```

Comment:
Can we retain the existing error for this and the keyspace?

```java
return Response.status(Response.Status.BAD_REQUEST)
                .entity(
                    new ApiError(
                        "table name must be provided", Response.Status.BAD_REQUEST.getStatusCode()))
                .build();
```

Reply:
Will do.